catalog/ui/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js

```diff
@@ -90,11 +106,15 @@ module.exports = Marionette.LayoutView.extend({
     },
     setupDropzone: function() {
         this.dropzone = new Dropzone(this.el.querySelector('.details-dropzone'), {
-            url: '/services/catalog/',
+            url: this.url,
             maxFilesize: 5000000, //MB
             method: 'post',
-            autoProcessQueue: false
+            autoProcessQueue: false,
+            headers: this.extraHeaders
```

Comment:
If we remove the storing off of the options above this would become `this.options.extraHeaders`.

Reply:
done